### ISSUE TITLE
Upgrade fails when updating from 2.8.25 to 2.9.13

### ISSUE THREAD

**Issue by ajburt98** [thorsten/phpMyFAQ#1550]
We are currently using the 2.8.25 of phpMyFAQ.  I tried to upgrade it to 2.9.13 this weekend, but the upgrade continually fails.  It gives the following error message:
Error: Please install your version of phpMyFAQ once again or send us a bug report.
DB error: 01000:[Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Executing SQL directly; no cursor
EXEC sp_RENAME 'faqdata.datum', 'updated', 'COLUMN'

I even tried looking at the upgrade script that phpMyFAQ uses and manually run the SQL statements.  However, after completing everything the sql scripts, I have the following issues:
1. Domain logins no longer work (I followed the upgrade suggestions and saved my LDAP.php)
2. Logging in to the user side of phpMyFAQ just produces a blank page.
3. Logging into the admin side does not show any articles.  However, if I search for them, they then show up but if you click on them to edit them, you get an access denied error even when logged in as an admin.

Here is my configuration:
- SQL Server 2016 - 13.0.1742.0 (resides on a separate server)
- The php database has a customized name
- The phpMyFAQ is currently running on PHP 5.6

Also, I have enabled the debug flag, but don't see anything indicating any issues.  

It acts like it is some sort of permissions issue after the upgrade.  Any help is appreciated.

<img width="1043" alt="Screen Shot 2019-03-12 at 7 43 43 PM" src="https://user-images.githubusercontent.com/40278883/54295575-b15cdd00-4589-11e9-81a6-9807c3384ad2.png">

**Comment by ajburt98** [comment]
I did validate that the fields that the scripts were trying to add or update worked.  For example, the faqdata.datum column did exist and if I ran the query manually, it did change it to "updated".  However, after I would run all of the scripts manually, the users for phpMyFAQ didn't have permission to access any of the articles.  If there is anything that you would like for me to try or if I can help in any way, please let me know.

**Comment by thorsten** [comment]
Okay, so the queries are okay, but according the error message there was "no cursor". Did you ran all queries in the update script from 2.9.0-alpha to 2.9.6?

**Comment by ajburt98** [comment]
Yes. I did run them all.  However, afterwards, when I tried to navigate to the login screen, it was just a  blank page. I then tried the admin section login and could not log in to that with my LDAP creds.  I could successfully log in to the admin side with a local user. Again, no articles would show.  However, if I searched for an article to edit, It would should the title of the article.  If I tried to click on it to edit, it would show access denied. I verified that the data is still all there in the database. It just seems like the permissions don’t work after the upgrade.


Jim Burt
Software Engineer
Common CENTS Solutions, Inc.
(601)326-8665 - Direct
(601)932-3687 - Office
6360 I-55 North, Suite 160
Jackson MS 39211

[cid:6B6DB0E7-2648-4972-920F-021BA13B8565]

On Mar 15, 2019, at 1:38 PM, Thorsten Rinne <notifications@github.com<mailto:notifications@github.com>> wrote:


Okay, so the queries are okay, but according the error message there was "no cursor". Did you ran all queries in the update script from 2.9.0-alpha to 2.9.6?
